### PR TITLE
chore(server): update data structure and sdk version

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/test/models/ckb_sync/api_test.rb [hunk 1]
```diff
@@ -31,18 +23,13 @@ class ApiTest < ActiveSupport::TestCase
...
-          }'
-          genesis_block = JSON.parse(fake_node_block, symbolize_names: true)
-          CKB::API.any_instance.stubs(:genesis_block).returns(genesis_block)
+          ENV["CKB_NODE_URL"] = "http://localhost:8114"
```
Comment (classicalliu) [inline server/test/models/ckb_sync/api_test.rb]: in `env.test` ?

File: server/app/models/cell_output.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class CellOutput < ApplicationRecord
   BASE_HASH = "0x0000000000000000000000000000000000000000000000000000000000000000".freeze
-  INITIAL_BLOCK_REWARD = 50_000
+  INITIAL_BLOCK_REWARD = 5_000_000_000_000
```
Comment (classicalliu) [inline server/app/models/cell_output.rb]: It should be a config

File: server/test/models/ckb_sync/api_test.rb (hunk 1 above)
Comment (shaojunda) [reply server/test/models/ckb_sync/api_test.rb]: thank you, this is not necessary, I will remove it.

File: server/app/models/cell_output.rb (hunk 2 above)
Comment (shaojunda) [reply server/app/models/cell_output.rb]: Tank you, it makes sense.

**Pull request merged**